rpmbuild/copr_rpmbuild/providers/pypi.py

```diff
@@ -56,7 +56,7 @@ def _produce_srpm_pyp2spec(self):
             "pyp2spec",
             self.pypi_package_name,
             "--fedora-compliant",
-            "--top-level",
```

Comment:
Why don't we need --top-level?

Reply:
It's included/hidden as implementation detail in the `automode` now.